lms/templates/course.html

```diff
@@ -6,34 +6,17 @@
 %>
 <%page args="course" />
 <article id="${course.id | h}" class="course">
-  %if course.is_newish:
-    <span class="status">${_("New")}</span>
-  %endif
   <a href="${reverse('about_course', args=[course.id.to_deprecated_string()])}">
-  <div class="inner-wrapper">
-      <header class="course-preview">
-        <hgroup>
-          <h2><span class="course-number">${course.display_number_with_default | h}</span> ${get_course_about_section(course, 'title')}</h2>
-        </hgroup>
-        <div class="info-link">&#x2794;</div>
-      </header>
-      <section class="info">
-        <div class="cover-image">
-          <img src="${course_image_url(course)}" alt="${course.display_number_with_default | h} ${get_course_about_section(course, 'title')} Cover Image" />
-        </div>
-        <div class="desc">
-          <p>${get_course_about_section(course, 'short_description')}</p>
-        </div>
-        <div class="bottom">
-          <span class="university">${get_course_about_section(course, 'university')}</span>
-          % if not course.start_date_is_still_default:
-          <span class="start-date">${course.start_datetime_text()}</span>
-          % endif
-        </div>
-      </section>
-    </div>
-    <div class="meta-info">
-      <p class="university">${get_course_about_section(course, 'university')}</p>
-    </div>
+    <header class="course-image">
+      <div class="cover-image">
+        <img src="${course_image_url(course)}" alt="${course.display_number_with_default | h} ${get_course_about_section(course, 'title')} Cover Image" />
```

Comment:
The fact that there is an image here is not important to the SR user, so an empty alt attribute is appropriate ```alt=""```, especially if the image is not going to be described to them (and it mostly likely shouldn't.  These are usually just eye-candy.